components/CharacterCard.tsx

```diff
@@ -160,12 +163,18 @@ export const CharacterCard: React.FC<{
                   Inventory ({itemTotal})
                 </Text>
               </HStack>
-              <Button variant="ghost" size="xs">
-                show all
-              </Button>
+              {items.length > 2 && (
+                <Button
+                  variant="ghost"
+                  size="xs"
+                  onClick={itemsCatalogModal.onOpen}
+                >
+                  show all
+                </Button>
+              )}
             </HStack>
             <SimpleGrid columns={2} spacing={4} w="full">
-              {items.map(item => (
+              {items.slice(0, 2).map(item => (
```

Comment:
Just pointing out again here that we should prob add the quipping action to the list of item actions, since we're now only showing 2 `ItemTag`s

Reply:
Added in f2b29eb9aeee29efbe30bf0dd5ea82ce0479c919